scripts/lxc-to-lxd

```diff
@@ -457,7 +532,7 @@ parser.add_argument("--all", action="store_true", default=False,
                     help="Import all containers")
 parser.add_argument("--delete", action="store_true", default=False,
                     help="Delete the source container")
-parser.add_argument("--copy-rootfs", action="store_true", default=False,
+parser.add_argument("--copy-rootfs", action="store_true", default=True,
```

Comment:
That's the second commit I added. From the commit message:

"This was hitting me by surprise when using this script. I'd argue that users
much rather expect a copy than a move. It also is more safe: If something goes
wrong users will still have their old container in place to recover."